x-pack/legacy/plugins/uptime/server/rest_api/create_route_with_auth.ts

```diff
@@ -15,21 +15,31 @@ export const createRouteWithAuth = (
 ): UMRouteDefinition => {
   const restRoute = routeCreator(libs);
   const { handler, method, path, options, ...rest } = restRoute;
-  const authHandler: RequestHandler<ObjectType, ObjectType, ObjectType> = async (
+  const licenseCheckHandler: RequestHandler<ObjectType, ObjectType, ObjectType> = async (
     context,
     request,
     response
   ) => {
-    if (libs.license(context.licensing.license)) {
+    const { statusCode, message } = libs.license(context.licensing.license);
+    if (statusCode === 200) {
       return await handler(context, request, response);
     }
-    return response.badRequest();
+    switch (statusCode) {
+      case 400:
+        return response.badRequest({ body: { message } });
+      case 401:
+        return response.unauthorized({ body: message });
+      case 403:
+        return response.forbidden({ body: message });
+      default:
+        return response.internalError();
```

Comment:
Two `({ body: message })` and one `({ body: { message } })`, is this expected?

Reply:
Thanks again... 💯

Reply:
Fixed in https://github.com/elastic/kibana/pull/51403/commits/ac4e2e9a62eb7d8069d60cd3c7b5eaa9126d3344.